### PR TITLE
Apply setup-py-upgrade

### PR DESCRIPTION
**Pull request by graingert**
Changes proposed in this pull request:

* apply setup-py-upgrade
* move setuptools_scm config to pyproject.toml

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+    Topic :: Text Processing
+    Topic :: Text Processing :: General
+
+[options]
+packages = find:
+package_dir = =src
+include_package_data = True
+zip_safe = False
+install_requires = setuptools
+setup_requires = setuptools_scm
```
Comment (graingert) [inline setup.cfg:39]:
```suggestion
```

**Comment by hugovk** [comment]
Thanks for this!

I've not used setup-py-upgrade before, but I'd like to use it on some other repos. Just to try it out, I ran it on `master` but it said:

```
$ setup-py-upgrade .
use_scm_version= is not supported in setup.cfg
```

Any tips?

---

Please could you also run setup-cfg-fmt and add it to pre-commit?

**Comment by graingert** [comment]
@hugovk I manually deleted that line and restored it

**Comment by graingert** [comment]
@hugovk I've now applied setup-cfg-fmt

**Comment by hugovk** [comment]
Thank you!

**Pull request merged**